lsusb.c

```diff
@@ -117,12 +117,14 @@
 
 unsigned int verblevel = VERBLEVEL_DEFAULT;
 static int do_report_desc = 1;
-static const char * const encryption_type[] = {
-	"UNSECURE",
+static const char *const encryption_type[] = {
+	// clang-format off
```

Comment:
That's not ok, sorry, if clang-format can't handle this, then don't use it.

It also doesn't have much to do with the codespell changes.